app/controllers/home_controller.rb

```diff
@@ -7,7 +7,7 @@ def get_components_structure
     @components = JSON.parse(result)['categories']
 
     @components.each_with_index do |component, index|
-        @components[index]['product_schema'] = JSON.parse(component['product_schema'])
+        @components[index]['product_schema'] = {"type": ['ssd', 'hdd'],"size": ['250gb', '500gb', '1tb']} # temporarily json object (mock)
```

Comment:
Can we rename `product_schema` to something else? `product_values` or something? Product schema is the json schema in the database that defines the product.